sdk/appconfiguration/feature-management/README.md

```diff
@@ -0,0 +1,236 @@
+            }
+        ]
+    },
+}
+```
+
+This object is passed into the `FeatureManager` when it is initialized.
+
+### Feature Filters
+
+A `FeatureFilter` is a class that can be used to enable/disable feature flags. Two built-in feature filters are provided:
+
+- `Microsoft.TimeWindowFilter` - Enables a feature flag based on a time window.
+- `Microsoft.TargetingFilter` - Enables a feature flag based on a list of users, groups, or rollout percentages.
+
+#### Time Window Filter
+
+The Time Window Filter enables a feature flag based on a time window. It has two parameters:
+
+- `Start` - The start time of the time window.
+- `End` - The end time of the time window.
+
+```json
+{
+    "name": "Microsoft.TimeWindowFilter",
+    "parameters": {
+        "Start": "2020-01-01T00:00:00Z",
+        "End": "2020-12-31T00:00:00Z"
+    }
+}
+```
+
+Both parameters are optional, but at least one is required. The time window filter is enabled after the start time and before the end time. If the start time is not specified, it is enabled immediately. If the end time is not specified, it will remain enabled after the start time.
+
+#### Targeting Filter
+
+Targeting is a feature management strategy that enables developers to progressively roll out new features to their user base. The strategy is built on the concept of targeting a set of users known as the target audience. An audience is made up of specific users, groups, excluded users/groups, and a designated percentage of the entire user base. The groups that are included in the audience can be broken down further into percentages of their total members.
+
+The following steps demonstrate an example of a progressive rollout for a new 'Beta' feature:
+
+1. Individual users Jeff and Alicia are granted access to the Beta
+1. Another user, Mark, asks to opt-in and is included.
+1. Twenty percent of a group known as "Ring1" users are included in the Beta.
+1. The number of "Ring1" users included in the beta is bumped up to 100 percent.
+1. Five percent of the user base is included in the beta.
+1. The rollout percentage is bumped up to 100 percent and the feature is completely rolled out.
+1. This strategy for rolling out a feature is built in to the library through the included Microsoft.Targeting feature filter.
+
+##### Defining a Targeting Feature Filter
+
+The Targeting Filter enables a feature flag based on a list of users, groups, or rollout percentages. It has a parameter:
+
+- `Audience` - The audience of the feature flag.
+
+Which contains the following fields:
+
+- `Users` - A list of users that the feature flag is enabled for.
+- `Groups` - A list of groups that the feature flag is enabled for, at different percentages.
+- `DefaultRolloutPercentage` - A percentage value that the feature flag is enabled for.
+- `Exclusions` - An object that contains a list of users and groups that the feature flag is disabled for.
+
+```json
+{
+    "name": "Microsoft.TargetingFilter",
+    "parameters": {
```

Comment:
Audience is missing from the example

Reply:
This should be fixed now, see the readme pr.